maps-ktx/src/main/java/com/google/maps/android/ktx/GoogleMap.kt

```diff
@@ -183,14 +184,15 @@ public fun GoogleMap.cameraMoveCanceledEvents(): Flow<Unit> =
     }
 
 /**
- * Returns a flow that emits when the camera moves. Using this to observe camera move events will
- * override an existing listener (if any) to [GoogleMap.setOnCameraMoveListener].
+ * Returns a flow that emits a [GoogleMap.getProjection] when the camera moves. Using this to
+ * observe camera move events will override an existing listener (if any) to
+ * [GoogleMap.setOnCameraMoveListener].
  */
 @ExperimentalCoroutinesApi
-public fun GoogleMap.cameraMoveEvents(): Flow<Unit> =
```

Comment:
>I think this should continue to return Flow<Unit> so that the return type matches that of the return type in the underlying callback (which is Unit). 

That makes sense.

>When consuming cameraMoveEvents(), can you instead map on the result and return a projection?

Hmmm, I'm not sure. What would this `map {}` look like? It's not clear to me how I'd map `Unit` to `GoogleMap.projection`.

Reply:
You would need to reference the `GoogleMap` from within the `map` function. 

```
googleMap.cameraMoveEvents().map { 
    googleMap.projection 
}
```

Reply:
Ah, thanks. Ok, so I tried exactly as above, but got this Lint error:
https://googlesamples.github.io/android-custom-lint-rules/checks/FlowOperatorInvokedInComposition.md.html

...which says:

>Calling a Flow operator function within composition will result in a new Flow being created every recomposition, which will reset collectAsState() and cause other related problems. Instead Flow operators should be called inside remember, or a side effect such as LaunchedEffect.

So in https://github.com/googlemaps/android-maps-ktx/pull/175/commits/395d924774127d617423338e18dfece795593f4d I've wrapped it inside `remember`, which I believe is the correct approach here (but feedback welcome):

```kotlin
    val projection by remember {
        googleMap.cameraMoveEvents().map {
            googleMap.projection
        }
    }.collectAsState(googleMap.projection)
```